src/main/java/macrobase/conf/MacroBaseConf.java

```diff
@@ -41,6 +41,8 @@
     public static final String MCD_ALPHA = "macrobase.analysis.mcd.alpha";
     public static final String MCD_STOPPING_DELTA = "macrobase.analysis.mcd.stoppingDelta";
 
+    public static final String NUM_MIXTURES = "macrobase.analysis.mixtures.numMixtures";
```

Comment:
that sounds good, but the convention so far is to skip `stats`. I'm fine changing it. But probably if we change we should change all, no?